lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -427,4 +438,33 @@ export class EvaluationClientWrapper {
     }
     return status;
   }
+
+  private updateSubmissionRequirementMatchPathToAlias(
+    submissionRequirementMatch: SubmissionRequirementMatch,
+    alias: string
+  ) {
+    const matches: string[] = [];
+    submissionRequirementMatch.matches.forEach((m) => {
+      matches.push(m.replace('$', '$.' + alias));
+    });
+    submissionRequirementMatch.matches = matches;
+    if (submissionRequirementMatch.from_nested) {
+      submissionRequirementMatch.from_nested.forEach((f) => {
+        this.updateSubmissionRequirementMatchPathToAlias(f, alias);
+      });
+    }
+  }
+
+  private updatePresentationSubmissionPathToAlias(alias: string) {
+    this._client.presentationSubmission.descriptor_map.forEach((d) => {
+      this.replacePathWithAlias(d, alias);
+    });
+  }
+
+  private replacePathWithAlias(descriptor: Descriptor, alias: string) {
+    descriptor.path = descriptor.path.replace('$', '$.' + alias);
+    if (descriptor.path_nested) {
+      this.replacePathWithAlias(descriptor.path_nested, alias);
```

Comment:
Does that make sense? A nested VC is not exposed at the root, but part of another VC right?

Reply:
same as above